aderyn_core/src/detect/low/sub_without_if.rs

```diff
@@ -0,0 +1,150 @@
+use std::collections::{BTreeMap, HashSet};
+use std::error::Error;
+
+use crate::ast::NodeID;
+
+use crate::capture;
+use crate::context::browser::ExtractMemberAccesses;
+use crate::context::investigator::{
+    StandardInvestigationStyle, StandardInvestigator, StandardInvestigatorVisitor,
+};
+
+use crate::context::workspace_context::ASTNode;
+use crate::detect::detector::IssueDetectorNamePool;
+use crate::{
+    context::workspace_context::WorkspaceContext,
+    detect::detector::{IssueDetector, IssueSeverity},
+};
+use eyre::Result;
+
+#[derive(Default)]
+pub struct SubWithoutIfDetector {
+    // Keys are: [0] source file name, [1] line number, [2] character location of node.
+    // Do not add items manually, use `capture!` to add nodes to this BTreeMap.
+    found_instances: BTreeMap<(String, usize, String), NodeID>,
+}
+
+impl IssueDetector for SubWithoutIfDetector {
```

Comment:
I don't quite understand which ticket this detector is related to. Could you link it?

Reply:
Check your messages